packages/apple-pay-integration/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
```

Comment:
Do we need `overrides` and `ignorePatterns` if nothing is different from the base config?

Reply:
I'll remove `overrides` and keep `ignorePatterns`, because we set `"ignorePatterns": ["**/*"]` in `../../.eslintrc.json`, which is in line with `sdk`.

Reply:
We should be adjusting that in sdk as well in that case. I will be cleaning that all up soon 😄